[2013-02-27 00:03:35,265][WARN ][cluster.service          ] [Golem] failed to reconnect to node [Golem][EMEdo6eiQMqP9X3vhE76RQ][inet[/172.16.200.239:9300]]
org.elasticsearch.transport.ConnectTransportException: [Golem][inet[/172.16.200.239:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:665)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: java.net.ConnectException: connection timed out
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processConnectTimeout(NioClientSocketPipelineSink.java:484)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:369)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2013-02-27 00:07:35,568][WARN ][cluster.service          ] [Golem] failed to reconnect to node [Golem][EMEdo6eiQMqP9X3vhE76RQ][inet[/172.16.200.239:9300]]
org.elasticsearch.transport.ConnectTransportException: [Golem][inet[/172.16.200.239:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:665)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: java.net.ConnectException: connection timed out
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processConnectTimeout(NioClientSocketPipelineSink.java:484)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:369)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2013-02-27 00:11:35,864][WARN ][cluster.service          ] [Golem] failed to reconnect to node [Golem][EMEdo6eiQMqP9X3vhE76RQ][inet[/172.16.200.239:9300]]
org.elasticsearch.transport.ConnectTransportException: [Golem][inet[/172.16.200.239:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:665)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: java.net.ConnectException: connection timed out
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processConnectTimeout(NioClientSocketPipelineSink.java:484)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:369)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2013-02-27 00:15:36,163][WARN ][cluster.service          ] [Golem] failed to reconnect to node [Golem][EMEdo6eiQMqP9X3vhE76RQ][inet[/172.16.200.239:9300]]
org.elasticsearch.transport.ConnectTransportException: [Golem][inet[/172.16.200.239:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:665)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: java.net.ConnectException: connection timed out
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processConnectTimeout(NioClientSocketPipelineSink.java:484)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:369)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2013-02-27 00:19:36,523][WARN ][cluster.service          ] [Golem] failed to reconnect to node [Golem][EMEdo6eiQMqP9X3vhE76RQ][inet[/172.16.200.239:9300]]
org.elasticsearch.transport.ConnectTransportException: [Golem][inet[/172.16.200.239:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:665)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: java.net.ConnectException: connection timed out
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processConnectTimeout(NioClientSocketPipelineSink.java:484)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:369)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2013-02-27 00:23:36,827][WARN ][cluster.service          ] [Golem] failed to reconnect to node [Golem][EMEdo6eiQMqP9X3vhE76RQ][inet[/172.16.200.239:9300]]
org.elasticsearch.transport.ConnectTransportException: [Golem][inet[/172.16.200.239:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:665)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: java.net.ConnectException: connection timed out
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processConnectTimeout(NioClientSocketPipelineSink.java:484)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:369)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2013-02-27 00:27:37,168][WARN ][cluster.service          ] [Golem] failed to reconnect to node [Golem][EMEdo6eiQMqP9X3vhE76RQ][inet[/172.16.200.239:9300]]
org.elasticsearch.transport.ConnectTransportException: [Golem][inet[/172.16.200.239:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:665)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: java.net.ConnectException: connection timed out
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processConnectTimeout(NioClientSocketPipelineSink.java:484)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:369)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2013-02-27 00:31:37,468][WARN ][cluster.service          ] [Golem] failed to reconnect to node [Golem][EMEdo6eiQMqP9X3vhE76RQ][inet[/172.16.200.239:9300]]
org.elasticsearch.transport.ConnectTransportException: [Golem][inet[/172.16.200.239:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:665)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: java.net.ConnectException: connection timed out
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processConnectTimeout(NioClientSocketPipelineSink.java:484)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:369)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2013-02-27 00:35:37,768][WARN ][cluster.service          ] [Golem] failed to reconnect to node [Golem][EMEdo6eiQMqP9X3vhE76RQ][inet[/172.16.200.239:9300]]
org.elasticsearch.transport.ConnectTransportException: [Golem][inet[/172.16.200.239:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:665)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: java.net.ConnectException: connection timed out
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processConnectTimeout(NioClientSocketPipelineSink.java:484)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:369)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2013-02-27 00:39:38,075][WARN ][cluster.service          ] [Golem] failed to reconnect to node [Golem][EMEdo6eiQMqP9X3vhE76RQ][inet[/172.16.200.239:9300]]
org.elasticsearch.transport.ConnectTransportException: [Golem][inet[/172.16.200.239:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:665)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: java.net.ConnectException: connection timed out
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processConnectTimeout(NioClientSocketPipelineSink.java:484)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:369)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2013-02-27 00:43:38,422][WARN ][cluster.service          ] [Golem] failed to reconnect to node [Golem][EMEdo6eiQMqP9X3vhE76RQ][inet[/172.16.200.239:9300]]
org.elasticsearch.transport.ConnectTransportException: [Golem][inet[/172.16.200.239:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:665)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: java.net.ConnectException: connection timed out
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processConnectTimeout(NioClientSocketPipelineSink.java:484)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:369)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2013-02-27 00:47:38,743][WARN ][cluster.service          ] [Golem] failed to reconnect to node [Golem][EMEdo6eiQMqP9X3vhE76RQ][inet[/172.16.200.239:9300]]
org.elasticsearch.transport.ConnectTransportException: [Golem][inet[/172.16.200.239:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:665)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: java.net.ConnectException: connection timed out
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processConnectTimeout(NioClientSocketPipelineSink.java:484)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:369)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2013-02-27 00:51:39,022][WARN ][cluster.service          ] [Golem] failed to reconnect to node [Golem][EMEdo6eiQMqP9X3vhE76RQ][inet[/172.16.200.239:9300]]
org.elasticsearch.transport.ConnectTransportException: [Golem][inet[/172.16.200.239:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:665)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: java.net.ConnectException: connection timed out
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processConnectTimeout(NioClientSocketPipelineSink.java:484)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:369)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2013-02-27 00:55:39,361][WARN ][cluster.service          ] [Golem] failed to reconnect to node [Golem][EMEdo6eiQMqP9X3vhE76RQ][inet[/172.16.200.239:9300]]
org.elasticsearch.transport.ConnectTransportException: [Golem][inet[/172.16.200.239:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:665)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: java.net.ConnectException: connection timed out
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processConnectTimeout(NioClientSocketPipelineSink.java:484)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:369)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2013-02-27 00:59:39,669][WARN ][cluster.service          ] [Golem] failed to reconnect to node [Golem][EMEdo6eiQMqP9X3vhE76RQ][inet[/172.16.200.239:9300]]
org.elasticsearch.transport.ConnectTransportException: [Golem][inet[/172.16.200.239:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:665)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: java.net.ConnectException: connection timed out
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processConnectTimeout(NioClientSocketPipelineSink.java:484)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:369)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2013-02-27 01:03:39,967][WARN ][cluster.service          ] [Golem] failed to reconnect to node [Golem][EMEdo6eiQMqP9X3vhE76RQ][inet[/172.16.200.239:9300]]
org.elasticsearch.transport.ConnectTransportException: [Golem][inet[/172.16.200.239:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:665)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: java.net.ConnectException: connection timed out
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processConnectTimeout(NioClientSocketPipelineSink.java:484)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:369)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2013-02-27 01:07:40,322][WARN ][cluster.service          ] [Golem] failed to reconnect to node [Golem][EMEdo6eiQMqP9X3vhE76RQ][inet[/172.16.200.239:9300]]
org.elasticsearch.transport.ConnectTransportException: [Golem][inet[/172.16.200.239:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:665)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: java.net.ConnectException: connection timed out
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processConnectTimeout(NioClientSocketPipelineSink.java:484)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:369)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2013-02-27 01:11:40,617][WARN ][cluster.service          ] [Golem] failed to reconnect to node [Golem][EMEdo6eiQMqP9X3vhE76RQ][inet[/172.16.200.239:9300]]
org.elasticsearch.transport.ConnectTransportException: [Golem][inet[/172.16.200.239:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:665)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: java.net.ConnectException: connection timed out
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processConnectTimeout(NioClientSocketPipelineSink.java:484)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:369)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2013-02-27 01:15:40,966][WARN ][cluster.service          ] [Golem] failed to reconnect to node [Golem][EMEdo6eiQMqP9X3vhE76RQ][inet[/172.16.200.239:9300]]
org.elasticsearch.transport.ConnectTransportException: [Golem][inet[/172.16.200.239:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:665)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: java.net.ConnectException: connection timed out
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processConnectTimeout(NioClientSocketPipelineSink.java:484)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:369)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2013-02-27 01:19:41,265][WARN ][cluster.service          ] [Golem] failed to reconnect to node [Golem][EMEdo6eiQMqP9X3vhE76RQ][inet[/172.16.200.239:9300]]
org.elasticsearch.transport.ConnectTransportException: [Golem][inet[/172.16.200.239:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:665)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: java.net.ConnectException: connection timed out
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processConnectTimeout(NioClientSocketPipelineSink.java:484)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:369)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2013-02-27 01:23:41,569][WARN ][cluster.service          ] [Golem] failed to reconnect to node [Golem][EMEdo6eiQMqP9X3vhE76RQ][inet[/172.16.200.239:9300]]
org.elasticsearch.transport.ConnectTransportException: [Golem][inet[/172.16.200.239:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:665)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: java.net.ConnectException: connection timed out
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processConnectTimeout(NioClientSocketPipelineSink.java:484)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:369)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2013-02-27 01:27:41,867][WARN ][cluster.service          ] [Golem] failed to reconnect to node [Golem][EMEdo6eiQMqP9X3vhE76RQ][inet[/172.16.200.239:9300]]
org.elasticsearch.transport.ConnectTransportException: [Golem][inet[/172.16.200.239:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:665)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: java.net.ConnectException: connection timed out
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processConnectTimeout(NioClientSocketPipelineSink.java:484)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:369)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2013-02-27 01:31:42,167][WARN ][cluster.service          ] [Golem] failed to reconnect to node [Golem][EMEdo6eiQMqP9X3vhE76RQ][inet[/172.16.200.239:9300]]
org.elasticsearch.transport.ConnectTransportException: [Golem][inet[/172.16.200.239:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:665)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: java.net.ConnectException: connection timed out
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processConnectTimeout(NioClientSocketPipelineSink.java:484)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:369)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2013-02-27 01:35:42,465][WARN ][cluster.service          ] [Golem] failed to reconnect to node [Golem][EMEdo6eiQMqP9X3vhE76RQ][inet[/172.16.200.239:9300]]
org.elasticsearch.transport.ConnectTransportException: [Golem][inet[/172.16.200.239:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:665)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: java.net.ConnectException: connection timed out
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processConnectTimeout(NioClientSocketPipelineSink.java:484)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:369)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2013-02-27 01:39:42,767][WARN ][cluster.service          ] [Golem] failed to reconnect to node [Golem][EMEdo6eiQMqP9X3vhE76RQ][inet[/172.16.200.239:9300]]
org.elasticsearch.transport.ConnectTransportException: [Golem][inet[/172.16.200.239:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:665)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: java.net.ConnectException: connection timed out
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processConnectTimeout(NioClientSocketPipelineSink.java:484)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:369)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2013-02-27 01:43:25,186][WARN ][transport.netty          ] [Golem] exception caught on transport layer [[id: 0xe4e4c076]], closing connection
java.net.SocketException: Network is unreachable
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:599)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.connect(NioClientSocketPipelineSink.java:496)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processSelectedKeys(NioClientSocketPipelineSink.java:452)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:365)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
[2013-02-27 01:43:25,188][WARN ][transport.netty          ] [Golem] exception caught on transport layer [[id: 0xf86d4513]], closing connection
java.net.SocketException: Network is unreachable
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:599)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.connect(NioClientSocketPipelineSink.java:496)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processSelectedKeys(NioClientSocketPipelineSink.java:452)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:365)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
[2013-02-27 01:43:25,189][WARN ][transport.netty          ] [Golem] exception caught on transport layer [[id: 0x9eef37d2]], closing connection
java.net.SocketException: Network is unreachable
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:599)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.connect(NioClientSocketPipelineSink.java:496)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processSelectedKeys(NioClientSocketPipelineSink.java:452)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:365)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
[2013-02-27 01:43:43,065][WARN ][cluster.service          ] [Golem] failed to reconnect to node [Golem][EMEdo6eiQMqP9X3vhE76RQ][inet[/172.16.200.239:9300]]
org.elasticsearch.transport.ConnectTransportException: [Golem][inet[/172.16.200.239:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:665)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: java.net.ConnectException: connection timed out
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processConnectTimeout(NioClientSocketPipelineSink.java:484)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:369)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2013-02-27 01:45:25,322][WARN ][transport.netty          ] [Golem] exception caught on transport layer [[id: 0xb07318c0]], closing connection
java.net.SocketException: Network is unreachable
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:599)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.connect(NioClientSocketPipelineSink.java:496)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processSelectedKeys(NioClientSocketPipelineSink.java:452)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:365)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
[2013-02-27 01:45:25,330][WARN ][transport.netty          ] [Golem] exception caught on transport layer [[id: 0xe64872e3]], closing connection
java.net.SocketException: Network is unreachable
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:599)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.connect(NioClientSocketPipelineSink.java:496)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processSelectedKeys(NioClientSocketPipelineSink.java:452)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:365)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
[2013-02-27 01:47:43,419][WARN ][cluster.service          ] [Golem] failed to reconnect to node [Golem][EMEdo6eiQMqP9X3vhE76RQ][inet[/172.16.200.239:9300]]
org.elasticsearch.transport.ConnectTransportException: [Golem][inet[/172.16.200.239:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:665)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: java.net.ConnectException: connection timed out
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processConnectTimeout(NioClientSocketPipelineSink.java:484)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:369)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2013-02-27 01:51:25,828][WARN ][transport.netty          ] [Golem] exception caught on transport layer [[id: 0x768f75e4]], closing connection
java.net.SocketException: Network is unreachable
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:599)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.connect(NioClientSocketPipelineSink.java:496)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processSelectedKeys(NioClientSocketPipelineSink.java:452)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:365)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
[2013-02-27 01:51:25,829][WARN ][transport.netty          ] [Golem] exception caught on transport layer [[id: 0x5585dc0d]], closing connection
java.net.SocketException: Network is unreachable
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:599)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.connect(NioClientSocketPipelineSink.java:496)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processSelectedKeys(NioClientSocketPipelineSink.java:452)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:365)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
[2013-02-27 01:51:25,830][WARN ][transport.netty          ] [Golem] exception caught on transport layer [[id: 0x971adebc]], closing connection
java.net.SocketException: Network is unreachable
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:599)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.connect(NioClientSocketPipelineSink.java:496)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processSelectedKeys(NioClientSocketPipelineSink.java:452)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:365)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
[2013-02-27 01:51:25,831][WARN ][transport.netty          ] [Golem] exception caught on transport layer [[id: 0x8a69b7fd]], closing connection
java.net.SocketException: Network is unreachable
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:599)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.connect(NioClientSocketPipelineSink.java:496)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processSelectedKeys(NioClientSocketPipelineSink.java:452)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:365)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
[2013-02-27 01:51:25,831][WARN ][transport.netty          ] [Golem] exception caught on transport layer [[id: 0xf8fdf0a1]], closing connection
java.net.SocketException: Network is unreachable
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:599)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.connect(NioClientSocketPipelineSink.java:496)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processSelectedKeys(NioClientSocketPipelineSink.java:452)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:365)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
[2013-02-27 01:51:43,765][WARN ][cluster.service          ] [Golem] failed to reconnect to node [Golem][EMEdo6eiQMqP9X3vhE76RQ][inet[/172.16.200.239:9300]]
org.elasticsearch.transport.ConnectTransportException: [Golem][inet[/172.16.200.239:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:665)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: java.net.ConnectException: connection timed out
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processConnectTimeout(NioClientSocketPipelineSink.java:484)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:369)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2013-02-27 01:54:46,121][WARN ][transport.netty          ] [Golem] exception caught on transport layer [[id: 0x05f8f9d1]], closing connection
java.net.SocketException: Network is unreachable
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:599)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.connect(NioClientSocketPipelineSink.java:496)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processSelectedKeys(NioClientSocketPipelineSink.java:452)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:365)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
[2013-02-27 01:55:44,123][WARN ][cluster.service          ] [Golem] failed to reconnect to node [Golem][EMEdo6eiQMqP9X3vhE76RQ][inet[/172.16.200.239:9300]]
org.elasticsearch.transport.ConnectTransportException: [Golem][inet[/172.16.200.239:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:665)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: java.net.ConnectException: connection timed out
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processConnectTimeout(NioClientSocketPipelineSink.java:484)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:369)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2013-02-27 01:59:44,467][WARN ][cluster.service          ] [Golem] failed to reconnect to node [Golem][EMEdo6eiQMqP9X3vhE76RQ][inet[/172.16.200.239:9300]]
org.elasticsearch.transport.ConnectTransportException: [Golem][inet[/172.16.200.239:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:665)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: java.net.ConnectException: connection timed out
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processConnectTimeout(NioClientSocketPipelineSink.java:484)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:369)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2013-02-27 02:03:44,766][WARN ][cluster.service          ] [Golem] failed to reconnect to node [Golem][EMEdo6eiQMqP9X3vhE76RQ][inet[/172.16.200.239:9300]]
org.elasticsearch.transport.ConnectTransportException: [Golem][inet[/172.16.200.239:9300]] connect_timeout[30s]
	at org.elasticsearch.transport.netty.NettyTransport.connectToChannels(NettyTransport.java:665)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:604)
	at org.elasticsearch.transport.netty.NettyTransport.connectToNode(NettyTransport.java:574)
	at org.elasticsearch.transport.TransportService.connectToNode(TransportService.java:127)
	at org.elasticsearch.cluster.service.InternalClusterService$ReconnectToNodes.run(InternalClusterService.java:395)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:680)
Caused by: java.net.ConnectException: connection timed out
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.processConnectTimeout(NioClientSocketPipelineSink.java:484)
	at org.elasticsearch.common.netty.channel.socket.nio.NioClientSocketPipelineSink$Boss.run(NioClientSocketPipelineSink.java:369)
	at org.elasticsearch.common.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
	at org.elasticsearch.common.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
	... 3 more
[2013-02-27 02:04:11,749][INFO ][node                     ] [Golem] {0.20.1}[1623]: stopping ...
[2013-02-27 02:04:24,834][INFO ][node                     ] [Golem] {0.20.1}[1623]: stopped
[2013-02-27 02:04:24,834][INFO ][node                     ] [Golem] {0.20.1}[1623]: closing ...
[2013-02-27 02:04:24,848][INFO ][node                     ] [Golem] {0.20.1}[1623]: closed
[2013-02-27 12:16:51,683][INFO ][node                     ] [Master Order] {0.20.1}[728]: initializing ...
[2013-02-27 12:16:51,719][INFO ][plugins                  ] [Master Order] loaded [], sites []
[2013-02-27 12:16:57,890][INFO ][node                     ] [Master Order] {0.20.1}[728]: initialized
[2013-02-27 12:16:57,891][INFO ][node                     ] [Master Order] {0.20.1}[728]: starting ...
[2013-02-27 12:16:58,173][INFO ][transport                ] [Master Order] bound_address {inet[/0:0:0:0:0:0:0:0%0:9300]}, publish_address {inet[/192.168.2.3:9300]}
[2013-02-27 12:17:01,446][INFO ][cluster.service          ] [Master Order] new_master [Master Order][i6-t1aQ9QgaLG906-STvEQ][inet[/192.168.2.3:9300]], reason: zen-disco-join (elected_as_master)
[2013-02-27 12:17:01,564][INFO ][discovery                ] [Master Order] elasticsearch/i6-t1aQ9QgaLG906-STvEQ
[2013-02-27 12:17:01,604][INFO ][http                     ] [Master Order] bound_address {inet[/0:0:0:0:0:0:0:0%0:9200]}, publish_address {inet[/192.168.2.3:9200]}
[2013-02-27 12:17:01,605][INFO ][node                     ] [Master Order] {0.20.1}[728]: started
[2013-02-27 12:17:03,440][INFO ][gateway                  ] [Master Order] recovered [1] indices into cluster_state
